docs/how-tos/provision-setup.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Complete setup for a machine"
+linkTitle: "Set a machine up"
+weight: 69
+type: "docs"
+description: "If you have received a machine that uses Viam and have been pointed to this guide, this guide will show you how to set it up."
+images: ["/platform/provisioning-demo.gif"]
+videos: ["/platform/provisioning-demo.webm", "/platform/provisioning-demo.mp4"]
+---
+
+If you have received a machine with Viam pre-installed on it, this guide will show you how to complete your device setup.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Set up your machine using the Viam mobile app](#set-up-your-machine-using-the-viam-mobile-app).
+1. [Set up your machine using the captive portal](#set-up-your-machine-using-the-captive-portal).
+1. [Update machine configuration](#update-machine-configuration) to update hotspot details or add additional networks.
+   {{< /alert >}}
+
+## Prerequisites
+
+- Physical hardware constituting a machine
+- A laptop or mobile device if using the Viam mobile app
+
+## Set up your machine using the Viam mobile app
```

Comment:
Should mention this is an either/or with the Captive Portal method below. Otherwise this reads like "step number 2 of 3" in the overall page.

Reply:
adding to the top of the page.